public/app/plugins/panel/alertlist/AlertInstances.tsx

```diff
@@ -36,8 +48,12 @@ export const AlertInstances = ({ alerts, options }: Props) => {
     (): Alert[] => filterAlerts(options, sortAlerts(options.sortOrder, alerts)) ?? [],
     [alerts, options]
   );
-
-  const hiddenInstances = alerts.length - filteredAlerts.length;
+  const isGrafanaAlert = grafanaTotalInstances !== undefined;
+  const hiddenInstances = isGrafanaAlert
```

Comment:
I'm not sure if I can parse this – can we simplify and pull these in to separate vars to make this a bit more readable?